apps/condo/pages/_app.tsx

```diff
@@ -527,44 +534,88 @@ const MyApp = ({ Component, pageProps }) => {
     )
 }
 
-/*
-    Configuration for `InMemoryCache` of Apollo
-    Add fields, related to pagination strategies of Apollo.
-    Items of some GraphQL global fields needs to be appended to list,
-    when paginated, rather than to be displayed as a slice of data, —
-    its like "Infinite scrolling" UI pattern. For example, fetching
-    more changes of a ticket on button click.
-    For those items, we need to set `concatPagination` strategy.
-    https://www.apollographql.com/docs/react/pagination/core-api/
- */
-const apolloCacheConfig: WithApolloProps['apolloCacheConfig'] = {
-    typePolicies: {
-        [BILLING_RECEIPT_SERVICE_FIELD_NAME]: {
-            // avoiding of building cache from ID on client, since Service ID is not UUID and will be repeated
-            keyFields: false,
-        },
-        BuildingSection: {
-            keyFields: false,
-        },
-        BuildingFloor: {
-            keyFields: false,
+const cacheConfig: InitCacheConfig = (cacheOptions) => {
```

Comment:
Why not to import it from `next/utils`?

Seems it will be hard to remember, that in new apps its placed in `utils`, and in old ones - in `pages/_app`